tapeout/src/main/scala/barstools/tapeout/transforms/GenerateTopAndHarness.scala

```diff
@@ -111,7 +112,7 @@ private class GenerateTopAndHarness(annotations: AnnotationSeq) extends LazyLogg
     val annos = new FirrtlStage().execute(Array.empty, generatorAnnotations)
     annos.collectFirst { case FirrtlCircuitAnnotation(circuit) => circuit } match {
       case Some(circuit) =>
-        dump(circuit, annos, topFir, harnessAnnoOut)
+        dump(circuit, annos, harnessFir, harnessAnnoOut)
```

Comment:
This looks like it will be a change in behavior, are people using this already using HarnessFirAnnotation?
Should harnessFir fall back to TopFirAnnotation if HarnessFirAnnotation is not present?

Reply:
This is a bug fix; the second pass is actually overwriting topFir. It should not fall back.